### PR TITLE
Get brands to auto suggest section

### PR DESCRIPTION
**Pull request by Samaamro20**
- View Brand page by Name

### CODE REVIEW

**Comment by Samaamro20** [comment]
@shahenazmonia could you see this?

File: client/src/components/Home/Autosuggests.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
 import React, {Component} from 'react';
 import './Autosuggests.css';
 import Autosuggest from 'react-autosuggest';
+<<<<<<< HEAD
```
Comment (shahenazmonia) [inline client/src/components/Home/Autosuggests.js]: solve the conflict and remove unusable lines

File: client/src/components/Home/Autosuggests.js [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+
+const brands = [
+  'Andy','Zara','ADAM','Ball','AA','AB','AC','BA','BB'
```
Comment (shahenazmonia) [inline client/src/components/Home/Autosuggests.js]: delete the static data, get the last version of the code from the master. This is not last update

File: client/src/components/Home/Autosuggests.js (hunk 1 above)
Comment (Samaamro20) [reply client/src/components/Home/Autosuggests.js]: Done

File: client/src/components/Home/Autosuggests.js (hunk 2 above)
Comment (Samaamro20) [reply client/src/components/Home/Autosuggests.js]: Done

File: client/src/components/Home/Autosuggests.js [hunk 3]
```diff
@@ -29,24 +36,21 @@ export default class Autosuggests extends Component {
...
       loading: false,
-      brandss:[]
+      statebrands:[]
     };
     componentDidMount() {
        axios
          .get("/brands")
          .then(({ data }) => {
-            this.brandss = data.slice();
+            this.statebrands = data;
```
Comment (shahenazmonia) [inline client/src/components/Home/Autosuggests.js]: why did use ```this.statebrands``` ? if you want to setState you should use ```this.setState()``` not ```this.statebrands```

File: client/src/components/Home/Autosuggests.js [hunk 4]
```diff
@@ -29,24 +36,18 @@ export default class Autosuggests extends Component {
...
-            this.brandss = data.slice();
             brands = data.map(brand => {
-             return {
-               id: brand.id,
-               name: brand.name
-             };
+              brands.push(brand.name);
+             return (brand.name)
            });
            this.setState({
```
Comment (shahenazmonia) [inline client/src/components/Home/Autosuggests.js]: You didn't use loading, delete it from the state. Also, you don't need to setState

Comment (Samaamro20) [reply client/src/components/Home/Autosuggests.js]: Done

File: client/src/components/Home/Autosuggests.js (hunk 3 above)
Comment (Samaamro20) [reply client/src/components/Home/Autosuggests.js]: Done

**Pull request merged**